### ISSUE TITLE
Ensure ARIA roles, states, and properties are valid

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2540]
**OCIO Violation**: Ensure ARIA roles, states, and properties are valid

**OCIO Notes:** 
This BUTTON has an aria-controls attribute value of 'full-width-tabpanel-0', which includes one or more invalid or duplicate ids
This BUTTON has an aria-controls attribute value of 'full-width-tabpanel-1', which includes one or more invalid or duplicate ids
This BUTTON has an aria-controls attribute value of 'full-width-tabpanel-2', which includes one or more invalid or duplicate ids
This DIV (role=combobox) must have all of the following attributes: aria-controls | aria-expanded
This DIV has an aria-labelledby attribute value of 'full-width-tab-0', which includes one or more invalid or duplicate ids

**OCIO priority:** 3.2
More info in [this sheet](https://doimspp.sharepoint.com/:x:/s/NRRD/Ea-CVbPsGDpDkDLSIe6kZS8BNeTJ7C8_M7yRS86t6VEU2g) or the [ocio/amp dashboard](https://departmentofinterior.levelaccess.us/public/reporting/prioritization_tool.php?report_id=1532)

**Instances** (by note)

1. This BUTTON has an aria-controls attribute value of 'full-width-tabpanel-0', which includes one or more invalid or duplicate ids

- [ ] https://revenuedata.doi.gov/how-revenue-works/    

2. This BUTTON has an aria-controls attribute value of 'full-width-tabpanel-1', which includes one or more invalid or duplicate ids

- [ ] https://revenuedata.doi.gov/how-revenue-works/    

3. This BUTTON has an aria-controls attribute value of 'full-width-tabpanel-2', which includes one or more invalid or duplicate ids

- [ ] https://revenuedata.doi.gov/how-revenue-works/    

4. This DIV (role=combobox) must have all of the following attributes: aria-controls | aria-expanded

- [ ] https://revenuedata.doi.gov/how-revenue-works/    
- [ ] https://revenuedata.doi.gov/explore/
- [ ] https://revenuedata.doi.gov/query-data

5. This DIV has an aria-labelledby attribute value of 'full-width-tab-0', which includes one or more invalid or duplicate ids

- [ ] https://revenuedata.doi.gov/how-revenue-works/   - Tootltips

**Comment by eelzi-ONRR** [comment]
2024 OCIO report

Instances: 346
Occurrence: 6%
Severity: 5
Priority: 2.7

**Comment by lpgoldstein** [comment]
No longer relevant
